jpsonic-main/src/main/java/com/tesshu/jpsonic/controller/SubsonicRESTController.java

```diff
@@ -1326,11 +1325,6 @@ public void getNowPlaying(HttpServletRequest req, HttpServletResponse response)
                 continue;
             }
 
-            UserSettings userSettings = securityService.getUserSettings(username);
-            if (!userSettings.isNowPlayingAllowed()) {
-                continue;
-            }
-
             long minutesAgo = status.getMinutesAgo();
             if (minutesAgo < LIMIT_OF_HISTORY_TO_BE_PRESENTED) {
```

Comment:
This code will be redesigned. Please confirm.

Reply:
Although it will be removed in this commit, similar controls will likely be added back in a subsequent pull request.



 - userSettings#NowPlayingAllowed will not be removed from the Database schema and will be reused. As shown in the code above, it is used in API controllers to control information disclosure.
 - The userSettings#NowPlayingAllowed setting is added to the user Settings page with a default of false.
   - #2691
 - By enabling the flag for the account that the Admin wants to make public, Nowplaying will be available just like regular Subsonic. 
   - This is not something that is decided by the user's preferences, but by the server administrator, taking into account the operational policy (just like other user settings). The main concerns for server administrators are server load and rules on how account name disclosure should be handled.